styles/style.css

```diff
@@ -0,0 +1,62 @@
+/* 
+
+nav {
+    display: inline;
+    float: right
+} */
+
+
+.profile-navigation {
+    display: block;
+    height: 44px;
+    background-color: #737373;
+    position: relative;
+    text-align: center;
+}
+
+.nav-li{
```

Comment:
Rather than creating a new class here, it would have been better to use the descendant combinator so this rule would be `nav li` which covers all `li` elements that are descendants of `nav` elements.

The same goes for the `.nav-a` rules below.